src/clients/python/experimental_api_v2/library/grpcclient.py

```diff
@@ -66,14 +68,15 @@ def _get_inference_request(model_name,
     for infer_output in outputs:
         request.outputs.extend([infer_output._get_tensor()])
     if sequence_id:
```

Comment:
Can you be explicit with these conditionals, use "is not None", != 0, etc. as appropriate.